superset-frontend/package.json

```diff
@@ -37,7 +37,7 @@
     "src/setup/*"
   ],
   "scripts": {
-    "_lint": "eslint --ignore-path=.eslintignore --ext .js,.jsx,.ts,tsx .",
+    "_lint": "eslint --ignore-path=.eslintignore --ext .js,.jsx,.ts,tsx",
```

Comment:
removing the `.` at the end so this private base command can be reused more generically